core/node/node_stats_tracker.go

```diff
@@ -52,6 +52,12 @@ type ProviderSessionsSeries func(id identity.Identity, rangeTime string) (Sessio
 // ProviderTransferredDataSeries should return transferred bytes data series metrics
 type ProviderTransferredDataSeries func(id identity.Identity, rangeTime string) (TransferredDataSeries, error)
 
+// ProviderActivityStats should return provider activity stats
+type ProviderActivityStats func(id identity.Identity) (ActivityStats, error)
+
+// ProviderQuality should return provider quality for some country
+type ProviderQuality func(id identity.Identity, countryCode string) (QualityInfo, error)
```

Comment:
I am not sure about `countryCode` - long time ago we had discussions about some sort of average quality of the provider node in the whole region